cluster/ubuntu/build.sh

```diff
@@ -54,10 +53,10 @@ cp $ETCD/etcd $ETCD/etcdctl binaries/master
 
 # k8s
 echo "Download kubernetes release ..."
-K8S_VERSION=${K8S_VERSION:-"1.0.3"}
+KUBE_VERSION=${KUBE_VERSION:-"1.0.3"}
```

Comment:
To propagate $KUBE_VERSION var which indicates a required release
https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-push.sh#L70